assets/app/views/edit/build-config.html

```diff
@@ -141,6 +142,54 @@
                             </div>
                           </div>
 
+                          <div ng-if="updatedBuildConfig.spec.strategy.type === 'JenkinsPipeline'">
+                            <h3>Jenkins Pipeline Configuration</h3>
+                            <div class="form-group">
+                              <label for="jenkinsfile-type">Jenkinsfile Type</label>
+                              <select
+                                id="jenkinsfile-type"
+                                class="form-control"
+                                ng-model="jenkinsfileOptions.type"
+                                ng-options="type.id as type.title for type in jenkinsfileTypes"
+                                aria-describedby="jenkinsfile-type-help">
+                              </select>
+                              <div class="help-block" id="jenkinsfile-type-help">
+                                Use a Jenkinsfile from the source repository or specify the
+                                Jenkinsfile content directly in the build configuration.
+                              </div>
+                            </div>
+
+                            <div ng-if="jenkinsfileOptions.type === 'path'" class="form-group">
+                              <label for="jenkinsfilePath">Jenkinsfile Source Path</label>
+                              <input class="form-control"
+                                     id="jenkinsfilePath"
+                                     name="jenkinsfilePath"
+                                     type="text"
+                                     ng-model="updatedBuildConfig.spec.strategy.jenkinsPipelineStrategy.jenkinsfilePath"
+                                     autocorrect="off"
+                                     autocapitalize="off"
+                                     spellcheck="false"
+                                     aria-describedby="jenkinsfile-path-help">
+                              <div class="help-block" id="jenkinsfile-path-help">
+                                Optional path to the Jenkinsfile realtive to the root of the source
```

Comment:
s/realative/relative/

and relative to the context dir, good catch @jwforres 

https://github.com/openshift/origin/pull/7949/files#diff-43dd40b4c2b85b388a3d9116729032e4R416